lib/Fhp/Action/GetStatementOfAccountXML.php

```diff
@@ -169,6 +169,6 @@ public function processResponse(Message $response)
             throw new UnsupportedException('More than 1 HICAZ response segment is not supported at the moment!');
         }
         // It seems that paginated responses, always contain a whole XML Document
-        $this->xml[] = $responseHicaz[0]->getGebuchteUmsaetze()->getData();
+        $this->xml = $responseHicaz[0]->getGebuchteUmsaetze()->getData();
```

Comment:
The brackets need to stay there, please make a foreach like I suggested.
The it is now it will break on paginated responses.